documentation/install_from_packages.markdown

```diff
@@ -1,16 +1,25 @@
 ---
 layout: default
-title: "Puppet Server: Installing From Packages"
+title: "Install Puppet Server"
 canonical: "/puppetserver/latest/install_from_packages.html"
 ---
 
 [repodocs]: https://puppet.com/docs/puppet/latest/puppet_platform.html
 
-Puppet Server is configured to use 2 GB of RAM by default. If you're just testing an installation on a Virtual Machine, this much memory is not necessary. To change the memory allocation, see [Memory Allocation](#memory-allocation).
+Puppet provides official packages that install Puppet Server 6.0 and all of its prerequisites on x86_64 architectures for the following platforms, as part of [Puppet Platform][repodocs].
 
-> If you're also using PuppetDB, check its [requirements](https://puppet.com/docs/puppetdb/latest/index.html#system-requirements).
+* Red Hat Enterprise Linux 6, 7
+* Debian 8 (Jessie), 9 (Stretch), 10 (Buster)
+* Ubuntu 18.04 (Bionic) - enable the [universe repository](https://help.ubuntu.com/community/Repositories/Ubuntu), which contains packages necessary for Puppet Server, 16.04 (Xenial)
```

Comment:
not understanding this note.. We have ubuntu 16.04 packages..

Reply:
I've moved 16.04 back to where this was. Is enabling the universe repository still needed for these versions?

Reply:
@clairecadman looks like yes, that is still necessary for ubuntu 18.04 (bionic).

I think it makes sense to move this note, though. It's pretty much the same as the one here: https://github.com/puppetlabs/puppetserver/pull/2361/files#diff-bb34594aa08d2f40dd4382784111dd7aR18
The reason this additional repository is necessary is because Java 8 runtime packages (openjdk-8-jre-headless) are not available in the standard repositories on ubuntu-18.04.